src/truststore/_windows.py

```diff
@@ -330,6 +330,13 @@ def _verify_peercerts_impl(
         chain_params.cbSize = sizeof(chain_params)
         pChainPara = pointer(chain_params)
 
+        if ssl_context.verify_flags & ssl.VERIFY_CRL_CHECK_CHAIN:
```

Comment:
Should we be starting with `chain_flags = 0` and then `|=` each Windows flag value onto `chain_flags` if the given `ssl.VERIFY_CRL_*` flag value is set?

In it's current state it looks like we can only either check the chain or the end cert, not both.

Reply:
The revocation check flags for the wincrypt APIs are mutually exclusive; we can set CERT_CHAIN_REVOCATION_CHECK_CHAIN or CERT_CHAIN_REVOCATION_CHECK_END_CERT but not both.

For both OpenSSL's flags and wincrypt's flags, it appears that CHECK_CHAIN also includes checking the leaf cert, so I think the way I've written this preserves the semantics without sending both flags to wincrypt at once.

Reply:
Got it, that makes sense! Thanks for clarifying.